src/AavePolygonTreasuryMigration_20230801/MIGRATE_POLYGON_V2_TREASURY_TO_POLYGON_V3.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Treasury Management - Polygon v2 to v3 Migration 
+discussions: https://governance.aave.com/t/arfc-migrate-consolidate-polygon-treasury/12248
+author: TokenLogic
+---
+
+# Summary
+
+This AIP shall redeem aTokens from Aave v2 Polygon and deposit the underlying assets into Aave v3 Polygon. 
+
+# Motivation
+
+On Polygon, Aave DAO is currently holding funds on the Aave v2 deployment and also some small unproductive allocations within the Treasury. This AIP will improve DAOs Treasury performance by improving the overall capital efficiency and yield being generated.
+
+As part of broader v2 to v3 migration effort, this AIP shall implement the following:
+* Redeem Assets from Aave v2
+* Deposit Assets from Aave v2 into v3
+
+* Deposit Assets held in Treasury into Aave v3
+
+For reference, the Aave Polygon Treasury: [`0xe8599F3cc5D38a9aD6F3684cd5CEa72f10Dbc383`](https://polygonscan.com/address/0xe8599F3cc5D38a9aD6F3684cd5CEa72f10Dbc383)
+
+# Implementation
```

Comment:
@defijesus I would also add a table containing the tokens which would be migrated with the amounts.